.github/workflows/ci-tests-pytorch.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       run: |
         python -m pip install -q pip -U
         extra=$(python -c "print({'lightning': 'pytorch-'}.get('${{ matrix.pkg-name }}', ''))")
-        pip install ".[${extra}extra,${extra}test]" -U \
+        pip install ".[${extra}extra,${extra}test,${extra}strategies]" -U \
```

Comment:
No this is meant without strategies

Reply:
agreed

Reply:
@awaelchli fabric cpu is installing strategies for deepspeed testing. Do you want that we remove that too?

Reply:
This was explicitly added in https://github.com/Lightning-AI/lightning/pull/16458. Why would the trainer ci be different?

Reply:
Ok, I was confused. 
Maybe we used to exclude them in the horovod days? It definitely used to have an impact back then.

Reply:
Yes:

```python
tests/tests_pytorch/utilities/test_imports.py:136:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_registry.py:41:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:106:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:123:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:139:@RunIf(deepspeed=True, mps=False)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:158:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:167:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:178:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:384:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:958:@RunIf(deepspeed=True, mps=False)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:1222:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:1275:@RunIf(deepspeed=True)
tests/tests_pytorch/strategies/test_deepspeed_strategy.py:1320:@RunIf(deepspeed=True)
tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py:241:        pytest.param("deepspeed", DeepSpeedStrategy, marks=RunIf(deepspeed=True)),
tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py:322:        pytest.param("deepspeed", DeepSpeedStrategy, marks=RunIf(deepspeed=True)),
tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py:360:        pytest.param("deepspeed", DeepSpeedStrategy, marks=RunIf(deepspeed=True)),
tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py:1000:        pytest.param("deepspeed", marks=RunIf(deepspeed=True)),
tests/tests_pytorch/plugins/test_cluster_integration.py:62:    [DDPStrategy, pytest.param(DeepSpeedStrategy, marks=RunIf(deepspeed=True))],
tests/tests_fabric/utilities/test_imports.py:41:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed_integration.py:240:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:45:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:53:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:62:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:73:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:82:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:98:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:124:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:145:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:155:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:163:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:185:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:208:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:222:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:242:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:252:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:275:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:294:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:318:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:347:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:367:@RunIf(deepspeed=True)
tests/tests_fabric/strategies/test_deepspeed.py:389:@RunIf(deepspeed=True, mps=False)
tests/tests_fabric/strategies/test_deepspeed.py:396:@RunIf(deepspeed=True)
tests/tests_fabric/plugins/precision/test_deepspeed_integration.py:24:@RunIf(deepspeed=True)
tests/tests_fabric/plugins/precision/test_deepspeed.py:38:@RunIf(deepspeed=True)
tests/tests_fabric/test_fabric.py:482:        pytest.param("deepspeed", marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_fabric.py:510:        pytest.param("deepspeed", marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_fabric.py:584:@RunIf(deepspeed=True, mps=False)
tests/tests_fabric/test_connector.py:287:        pytest.param("deepspeed", DeepSpeedStrategy, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:305:        pytest.param("deepspeed", DeepSpeedStrategy, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:421:        pytest.param("deepspeed", DeepSpeedStrategy, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:796:        pytest.param("32-true", "deepspeed", DeepSpeedPrecision, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:797:        pytest.param("16-true", "deepspeed", DeepSpeedPrecision, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:798:        pytest.param("bf16-true", "deepspeed", DeepSpeedPrecision, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:799:        pytest.param("16-mixed", "deepspeed", DeepSpeedPrecision, marks=RunIf(deepspeed=True)),
tests/tests_fabric/test_connector.py:800:        pytest.param("bf16-mixed", "deepspeed", DeepSpeedPrecision, marks=RunIf(deepspeed=True)),
```